core/src/main/java/org/elasticsearch/action/bulk/TransportShardBulkAction.java

```diff
@@ -135,7 +135,7 @@ private static BulkItemResultHolder executeIndexRequest(final IndexRequest index
         if (indexResult.hasFailure()) {
             return new BulkItemResultHolder(null, indexResult, bulkItemRequest);
         } else {
-            IndexResponse response = new IndexResponse(primary.shardId(), indexRequest.type(), indexRequest.id(),
+            IndexResponse response = new IndexResponse(primary.shardId(), indexRequest.type(), indexRequest.id(), indexRequest.source().utf8ToString(),
```

Comment:
@nik9000 this pr is for an assignment, due to another pr being open for this issue I have decided to return the request instead as duplicated code won't be accepted. Thanks for your help and sorry for any inconvenience